### PR TITLE
Expand tripleo deploy target with overcloud bits

### PR DESCRIPTION
**Pull request by marios**
As part of [1] this adds the required config and scripts for the overcloud.
It replaces scripts/undercloud.sh with /tripleo.sh which includes the
undercloud and overcloud deployment steps.

https://issues.redhat.com/browse/OSPRH-5278

[1] https://issues.redhat.com/browse/OSPRH-3038

### CODE REVIEW

File: devsetup/tripleo/config-download.yaml [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    compute-0-ctlplane:
+      fixed_ips:
+        - ip_address: 192.168.122.106
+      subnets:
+        - cidr: 192.168.122.0/24
+      network:
+        tags:
+          - 192.168.122.0/24
+  NodePortMap:
+    overcloud-controller-0:
```
Comment (marios) [inline devsetup/tripleo/config-download.yaml:55]: use same as hostname map eg np0004531736   np0004531737     np0004531738

**Comment by morucci** [comment]
recheck

**Comment by morucci** [comment]
recheck

**Comment by morucci** [comment]
recheck

**Comment by marios** [comment]
> recheck

@morucci o/ why the rechecks (i am actually using this in testproject to run a new adoption job that I am developing)

**Review by cescgina — APPROVED** [review]
lgtm

**Pull request merged**